pages/common/distccd.md

```diff
@@ -0,0 +1,24 @@
+# distccd
+
+> Server daemon for distcc distributed compiler.
+> More information: <https://distcc.github.io>.
+
+- Start a daemon with default settings:
+
+`distccd --daemon`
+
+- Start a daemon, accepting connections from private network IPv4 ranges:
+
+`distccd --daemon --allow-private`
+
+- Start a daemon, accepting connections from a specific network address or address range:
+
+`distccd --daemon --allow {{IP[/BITS]}}`
```

Comment:
We don´t use this notation `IP[/BITS]`, we use `lower_snake_case` and ORs `|` for argument options:
```suggestion
`distccd --daemon --allow {{ip_address|netmask_address}}`
```

Reply:
This bit is taken straight from `distccd --help` output.
`    -a, --allow IP[/BITS]      client address access control`
I thought the original output would be clearer, but if it violates tldr guidelines, I am in no opposition.

Reply:
:thinking: With your example I think I can enter a specific IP like `192.168.0.4` or a mask like `192.168.0.0/24`. Is this correct?

Reply:
Yes, that's my understanding as well. Personally my knowledge about networking is quite limited, and perhaps that's the reason why `IP[/BITS]` is easier for me to understand.
We may settle this by splitting this example into two. One example for a specific IP, and another for IP with BITS. ;)

Reply:
Sounds good, @tomek-szczesny!

Indeed, a netmask is not what we're after here - an example of a netmask is `255.255.255.0`.